src/analysis/convex/topology.lean

```diff
@@ -204,6 +205,39 @@ have hf : continuous (function.uncurry f),
 show f x y ∈ closure s,
   from map_mem_closure₂ hf hx hy (λ x' hx' y' hy', hs hx' hy' ha hb hab)
 
+open affine_map
+
+/-- A convex set `s` is strictly convex provided that for any two distinct points of
+`s \ interior s`, the line passing through these points has nonempty intersection with
+`interior s`. -/
+lemma convex.to_strict_convex' {s : set E} (hs : convex 𝕜 s)
```

Comment:
```suggestion
lemma convex.strict_convex' {s : set E} (hs : convex 𝕜 s)
```
The `to_` convention is reserved for instance names.

Reply:
We already had `convex.strict_convex`. I renamed the old lemma to `convex.strict_convex_of_open` and golfed lemmas about `strict_convex_Ixx`.